src/libcore/num/mod.rs

```diff
@@ -749,21 +749,18 @@ $EndFeature, "
             }
         }
 
-        doc_comment! {
-            concat!("Unchecked integer addition. Computes `self + rhs, assuming overflow
-cannot occur. This results in undefined behavior when `self + rhs > ", stringify!($SelfT),
-"::max_value()` or `self + rhs < ", stringify!($SelfT), "::min_value()`."),
-            #[unstable(
-                feature = "unchecked_math",
-                reason = "niche optimization path",
-                issue = "none",
-            )]
-            #[must_use = "this returns the result of the operation, \
-                          without modifying the original"]
-            #[inline]
-            pub unsafe fn unchecked_add(self, rhs: Self) -> Self {
-                intrinsics::unchecked_add(self, rhs)
-            }
+        /// Unchecked integer addition. Computes `self + rhs`, assuming overflow
+        /// cannot occur. This results in undefined behavior when `self + rhs > Self::MAX`
```

Comment:
I would prefer to keep the correct types in this documentation as they were before, e.g. `u8::MAX` rather than `Self::MAX`. The latter isn't as good in the context of the user's code since `Self::MAX` isn't a thing they can write in their code.

Same comment on all other documentation changed in this PR.